client.go

```diff
@@ -62,13 +62,14 @@ type Params struct {
 // Client represents the Confluent SDK client.
 type Client struct {
 	*Params
-	sling      *sling.Sling
-	Account    AccountInterface
-	Auth       Auth
-	Signup     Signup
-	Billing    Billing
-	LoginRealm LoginRealm
-	Growth     Growth
+	sling            *sling.Sling
+	Account          AccountInterface
```

Comment:
Unrelated to this PR, but could we make this change?
```suggestion
	Account          Account
```

Reply:
can't.. there's a struct called `account`. they were in cc-structs so didn't matter